site/docs/v1/tech/apis/api-explorer.adoc

```diff
@@ -0,0 +1,78 @@
+---
+layout: doc
+title: API Explorer
+description: Explore the API for FusionAuth
+---
+
+++++
+<link rel="stylesheet" type="text/css" href="./swagger-ui.css" />
+<style>
+
+/* overrides docs css that causes code text to be dark */
+code.language-json  p {
+color: #FFFFFF;
+}
+code.language-json  span {
+color: #FFFFFF;
+}
+code.language-bash  p {
+color: #FFFFFF;
+}
+code.language-bash  span {
+color: #FFFFFF;
+}
+
+pre.microlight span.headerline {
+color: #FFFFFF;
+}
+
+/* hides smartbear image */
+.topbar-wrapper a.link {
+display:none;
+}
+</style>
+++++
+
+== Overview
+
+This tool uses the https://github.com/fusionauth/fusionauth-openapi[FusionAuth OpenAPI specification] to let you explore APIs FusionAuth offers.
+
+* <<Configuration>>
+** <<In-Browser API Calls>>
+* <<Explorer>>
+
+== Configuration
+
+This tool defaults to using the latest version of the FusionAuth API. If you are using a different version, find the https://github.com/FusionAuth/fusionauth-openapi/tags[correct version] and update the URL in the input box, then click [field]#Explore#.
+
+For example, to view the 1.40.0 release, put `https://raw.githubusercontent.com/FusionAuth/fusionauth-openapi/1.40.0/openapi.yaml` in the YAML file input box.
+
+Don't forget to click on the [field]#Authorize# button to add your FusionAuth API key.
```

Comment:
Do you have more info on how the API key is handled? Does swagger store the potentially very powerful API key in browser storage or something? Is it cleared on page refresh?

Reply:
It is cleared on page refresh, per the default configuration: https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/#authorization

Looks like if you change it to persist authorization, it uses localStorage: https://github.com/swagger-api/swagger-ui/blob/8f63462f9eb634b9319a8be7cfe7090fbe8e304c/src/core/plugins/auth/actions.js#L274